underfs/hdfs/src/main/java/alluxio/underfs/hdfs/activesync/SupportedHdfsActiveSyncProvider.java

```diff
@@ -341,7 +337,7 @@ public SyncInfo getActivitySyncInfo() {
           syncSyncPoint(uri.toString());
         }
         mEventMissed = false;
-        LOG.debug("force syncPoint");
+        LOG.debug("Missed event, syncing all sync points");
```

Comment:
Could you combine these two messages, they are not guaranteed to be outputted together.